src/Microsoft.Health.Fhir.Core/Features/Persistence/ResourceRequest.cs

```diff
@@ -21,7 +21,7 @@ public ResourceRequest(string method, Uri url = null)
         {
             EnsureArg.IsNotNullOrEmpty(method, nameof(method));
 
-            Url = url;
+            Url = url != null ? null : null;
```

Comment:
Is this intended to always assign null? Or did you mean url != null ? url : null;